src/clients/authorization/Permissions.tsx

```diff
@@ -166,10 +170,12 @@ export const AuthorizationPermissions = ({ clientId }: PermissionsProps) => {
     return <KeycloakSpinner />;
   }
 
+  const noData = permissions.length === 0;
+  const searching = !isEqual(search, {});
```

Comment:
I think you could accomplish the same without using Lodash by doing the following:
```suggestion
  const searching = Object.keys(search).length !== 0;
```